CHANGELOG.md

```diff
@@ -11,6 +11,13 @@ and this project adheres to the versioning scheme outlined in the [README.md](RE
 
 ### Changed
 
+## [3.0.0.0.4]
+
+### Added
+
+### Changed
+
+- Load the BurnStateDB implementation from the block's header's `consensus_hash` field
```

Comment:
I might just use the comment from the commit message:

> use the same burn view loader in both block validation and block processing

Reply:
updated